### PR TITLE
Add glyphs U+2032..U+2033, rename U+2034

### PR DESCRIPTION
**Pull request by jublo**
TODO:

- [x] confirm that "millisecond" glyph is rendered with new name change

### CODE REVIEW

**Comment by chrissimpkins** [comment]
Confirming that we are breaking build because you renamed a *.glif file?

**Comment by jublo** [comment]
@chrissimpkins Can you update the metadata accordingly?

**Comment by chrissimpkins** [comment]
Yes no problem.  ufolint seems to indicate that it is unhappy because a glif file that is named in contents.plist is missing.  It appears that this is the problem.  Is "millisecond" the way that Glyphs names this file by default?

File: source/Hack-Bold.ufo/glyphs/millisecond.glif [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<glyph name="uni2034" format="1">
+<glyph name="millisecond" format="1">
   <advance width="1233"/>
   <unicode hex="2034"/>
   <outline>
```
Comment (chrissimpkins) [inline source/Hack-Bold.ufo/glyphs/millisecond.glif]: It looks like each of the "millisecond" glyphs are missing outline definitions.  Are they rendering the glyph for you?

**Comment by chrissimpkins** [comment]
Updated your OP with a TODO for this once we get to a build that works.  Mind checking this locally by replacing the uni2034 in your `contents.plist` files with "millisecond", building, and checking that you can display the glyph with the fonts?

**Comment by chrissimpkins** [comment]
No need to push the contents.plist.  I can make the same change so that we don't reorder it but it would be helpful to know that your changes render.  If they do not we should change the name back to the unicode version.

**Comment by jublo** [comment]
After replacing `uni2034` with `millisecond` in all 4 `contents.plist` files and then compiling, Font Book says this:

grafik

**Comment by chrissimpkins** [comment]
The source you committed here does not have a shape defined for those glyphs.  Are they unstaged changes in your local repo?

**Comment by jublo** [comment]
grafik

grafik

grafik

grafik

**Comment by chrissimpkins** [comment]
Apologies.  For some reason, Github was folding the source files in the diff.  I didn't see the source lines.  The outlines are there.  Should be gtg

**Comment by chrissimpkins** [comment]
Thanks for working on all of these @jublo

**Comment by chrissimpkins** [comment]
Merged!  Thanks @jublo !

**Pull request merged**